src/Symfony/Component/Process/Process.php

```diff
@@ -1099,15 +1103,7 @@ public function getEnv()
      */
     public function setEnv(array $env)
     {
-        // Process can not handle env values that are arrays
```

Comment:
We need to trigger a deprecation warning when passing array values (this was added because some people were passing ``$_SERVER`` directly at some point, when inheriting was not a native feature).